tutorials/javaFxPart2.md

```diff
@@ -22,6 +22,8 @@ Mockup | Control
 ![TextField](images/javafx/MockupTextField.png) | TextField
 ![ScrollPane](images/javafx/MockupScrollPane.png) | ScrollPane
 
+:bulb: You may be wondering why a ScrollPane is a `Control`, and not a `Pane`, as its name suggests. `ScrollPane` extends the `Control` class in JavaFX, because it represents a UI "widget" which the user directly interacts with (such as `Label`, `Button`, and `TextField`), whereas a `Pane` is a mere container that contains the other components. Of course, ScrollPane has the functionality of a container, allowing it to contain, or rather show a viewport of, its content.
```

Comment:
```suggestion
{{ icon_tip }} You may be wondering why a ScrollPane is a `Control`, and not a `Pane`, as its name suggests. `ScrollPane` extends the `Control` class in JavaFX, because it represents a UI "widget" which the user directly interacts with (such as `Label`, `Button`, and `TextField`), whereas a `Pane` is a mere container that contains the other components. Of course, ScrollPane has the functionality of a container, allowing it to contain, or rather show a viewport of, its content.
```

For better consistency.

Reply:
Sure! Thank you prof. A new commit is made.